.github/workflows/fastapi-test.yaml

```
@@ -55,14 +77,14 @@ jobs:
 
 
 
-      - name: Decode json credentials
-        run: echo ${{ secrets.SMARTAPI_CREDENTIALS }} | jq > smartapi_credentials.json
+      # - name: Decode json credentials
+      #   run: echo ${{ secrets.SMARTAPI_CREDENTIALS }} | jq > smartapi_credentials.json
 
      # Set up an environment variable with value "app"
       - name: Set up environment variable
         run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:19: Double quote to prevent globbing and word splitting

(shellcheck)